requirements.txt

```diff
@@ -9,3 +9,5 @@ h5py
 urllib3
 astroquery
 exo-det-box
+coverage
```

Comment:
this is not a core requirement for installing and operating EXOSIMS. (the general end user doesn't run coverage or end-to-end tests unless we ask them to for debugging purposes)

Reply:
Okay- I'm assuming this'll be the same with unittest-xml-runner as well.